RELEASE_NOTES.md

```diff
@@ -1,6 +1,11 @@
 Search Service MKII release notes
 =================================
 
```

Comment:
I realize I need a primer on how the recursive indexing works.

1) Can the recursive indexing target a subobject for data extraction, or only the parent object?
2) Can ResolvedRefs ever contain GUIDs with subobject IDs? I don't think they're every produced like that from the event handler, but I don't think that behavior is documented.

Reply:
I need a primer every time I look at that bit of code. I think recursive indexing does target subobjects. I had to put in the subobject id into MOPProvider.getObjectsByIds() method to get it all to work and I forget now the exact reason for this. I'd need to look it up again :(